package.json

```diff
@@ -39,6 +39,7 @@
     "sinon": "1.10.3",
     "sinon-chai": "2.5.0",
     "bower": "1.3.12",
-    "grunt-sass": "0.17.0"
+    "grunt-sass": "0.17.0",
+    "underscore": "1.8.3"
   }
```

Comment:
underscore/lodash were not npm dependencies before, so we aren't overriding anything in use, but yeah the matrix test will override underscore with the various versions and with lodash.